src/keymap.c

```diff
@@ -3452,8 +3402,7 @@ be preferred.  */);
   where_is_cache = Qnil;
   staticpro (&where_is_cache);
   staticpro (&where_is_cache_keymaps);
-  
```

Comment:
See those `staticpro`? That protects the variable from the garbage collector (aka GC). Checkout PR #641

Reply:
I see. So it seems I need to use `declare_GC_protected_static` to protect the variable. Alright!

Reply:
Correct. Remove the remaining C bits and update the Rust.

Reply:
Once you are confident in this PR, would you update the porting section of the wiki?